packages/documentation/pages/foundations/colors/theming.vue

```diff
@@ -53,7 +53,7 @@ Stylise the navbar with the select fields:
 
 <script>
 export default {
-	name: 'Theming',
+	name: 'CustomTheming',
```

Comment:
Should follow the naming strategy the other doc pages follow (something like `DocumentationPageFoundationsColor...`)